src/corelibs/U2Algorithm/src/util_msa_consensus/MSAConsensusAlgorithmLevitsky.h

```diff
@@ -74,7 +74,7 @@ class U2ALGORITHM_EXPORT MSAConsensusAlgorithmLevitsky : public MSAConsensusAlgo
     virtual MSAConsensusAlgorithmLevitsky* clone() const;
 
 private:
-    QVarLengthArray<int> globalFreqs;
+    QVarLengthArray<int> globalFreqs = QVarLengthArray<int>(256);
```

Comment:
Is there any reason for this change? Why not to assing QVarLengthArray<int>(256) directly in the constructor?